src/ExtendedXmlSerializer/IExtendedXmlSerializerConfig.cs

```diff
@@ -27,12 +27,16 @@ namespace ExtendedXmlSerialization
 {
     public interface IExtendedXmlSerializerConfig
     {
+        [Obsolete( "This property is no longer used.")]
```

Comment:
I think this property should be removed or still used. I approve it now but it will be changed.